src/websites/schemas/UpdateWebsite.schema.ts

```diff
@@ -2,6 +2,55 @@ import { WebsiteStatus } from '@prisma/client'
 import { VanityPathSchema } from './VanityPath.schema'
 import { createZodDto } from 'nestjs-zod'
 import { z } from 'zod'
+import { GOOGLE_ADDRESS_PLACE_VALUES } from 'prisma/schema/website.jsonTypes'
+
+const GoogleAddressComponentSchema = z.object({
+  long_name: z.string(),
+  short_name: z.string(),
+  types: z.array(z.enum(GOOGLE_ADDRESS_PLACE_VALUES)),
+})
+
+const GooglePlaceGeometrySchema = z.object({
+  location: z
+    .object({
+      lat: z.number(),
+      lng: z.number(),
+    })
+    .optional(),
+  viewport: z
+    .object({
+      south: z.number().optional(),
+      west: z.number().optional(),
+      north: z.number().optional(),
+      east: z.number().optional(),
+    })
+    .optional(),
+})
+
+const GooglePlusCodeSchema = z.object({
+  compound_code: z.string(),
+  global_code: z.string(),
+})
+
+const GooglePlacesApiResponseSchema = z.object({
+  address_components: z.array(GoogleAddressComponentSchema),
+  adr_address: z.string(),
+  formatted_address: z.string(),
+  geometry: GooglePlaceGeometrySchema.optional(),
+  icon: z.string().optional(),
+  icon_background_color: z.string().optional(),
+  icon_mask_base_uri: z.string().optional(),
+  name: z.string(),
+  place_id: z.string(),
+  plus_code: GooglePlusCodeSchema.optional(),
+  reference: z.string().optional(),
+  types: z.array(z.enum(GOOGLE_ADDRESS_PLACE_VALUES)),
+  url: z.string().optional(),
+  utc_offset: z.union([z.number(), z.string()]).optional(),
+  vicinity: z.string().optional(),
+  html_attributions: z.array(z.string()).optional(),
+  utc_offset_minutes: z.union([z.number(), z.string()]).optional(),
+})
```

Comment:
I'm wondering if these, along w/ the underlying GooglePlaces types in `website.jsonTypes` should just live in a `GoogleModule` to be importable in other modules... 🤔 

I struggled a bit w/ doing all this in my last PR since it doesn't appear to be anything out there for idiomatic server-side typing for GooglePlaces that _matches_ w/ Google's ui-side of things. So ultimate I decided not to do so.

So I'm glad you did this instead lol.. But now I'm thinking my code in the `TCRCompliance` stuff could benefit from it as well. But w/o having to import the entire `WebsiteModule`.